stock_fleet_delivery_driver/models/delivery_carrier.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    vehicle_id = fields.Many2one("fleet.vehicle", string="Vehicle")
```

Comment:
Change delivery driver when vehicle changes (put Vehicle on the first place in the form view)